src/screens/TreeDetailsScreen.tsx

```diff
@@ -79,6 +100,17 @@ export default function TreeDetailsScreen({
         value={entry.id}
       />
       {isEditing && <Button title="Save Changes" onPress={handleSaveChanges} />}
```

Comment:
what is the point of having a "save changes" button? It looks like it onyl appears sometimes -- and pressing "add comment" still works without it